app/controllers/users_controller.rb

```diff
@@ -1,29 +1,77 @@
 class UsersController < ApplicationController
+  before_action :logged_in_user, only: [:index, :edit, :update, :destroy]
```

Comment:
```suggestion
  before_action :logged_in_user, only: %i(index edit update destroy)
```
sửa những cái còn lại nữa nhé